src/pages/EditQuestion.jsx

```diff
@@ -53,15 +63,21 @@ export default function EditQuestion({ question, onUpdated }) {
 
   useEffect(() => { fetchHistoricAnswers(); }, [user]);
 
+  function questionFileName() {
+    const question_name_slug = slugify(question.metadata.name, '_',
+      { strict: true });
+    return `${question_name_slug}.json`;
+  }
+
   return (
     <Box mx={1}>
 
-      <Box my={3} width={1 / 2}>
+      <Box my={3}>
         <TextField
           fullWidth
           value={newQuestion.metadata.name}
           onChange={(e) => updateNewQuestion({ ...newQuestion, metadata: { ...newQuestion.metadata, name: e.target.value } })}
-          InputProps={{ readOnly: !question.owned }}
+          InputProps={{ disabled: !question.owned }}
```

Comment:
Since you're already making changes in this file, please clean this up to just be `disabled={!question.owned}`. Same for the select below.